src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -61,116 +63,45 @@ public void basicMarbleRoll(final String gtfFileName,
         runCommandLine(arguments);
     }
 
-    @Test(dataProvider = "provideDataForBasicMarbleRoll")
-    public void exhaustiveArgumentTest(final String gtfFileName,
+    @Test(dataProvider = "provideForIntegrationTest")
+    public void exhaustiveArgumentTest(final String dataSourcesPath,
+                                       final FuncotatorArgumentDefinitions.ReferenceVersionType refVer,
                                        final String referenceFileName,
-                                       final String fastaFileName,
                                        final String variantFileName,
                                        final String transcriptName,
-                                       final SimpleKeyXsvFuncotationFactory.XsvDataKeyType xsvMatchType,
-                                       final int xsvMatchColumn) throws IOException {
+                                       final SimpleKeyXsvFuncotationFactory.XsvDataKeyType xsvMatchType) throws IOException {
 
-        final String outFileName = "funcotator_tmp_out_" + xsvMatchType.toString() + "_" + xsvMatchColumn + "_" + transcriptName + ".vcf";
+        final String outFileName = "funcotator_tmp_out_" + xsvMatchType.toString() + "_" + transcriptName + ".vcf";
```

Comment:
For all automated tests, please write output files to the tmp dir.  Otherwise, you clutter up the source tree and people accidentally commit these to the repo..

Reply:
Yeah - I'm using `IOUtils.createTempFile` so that shouldn't be a problem.